### DISCUSSION TITLE
Q: Deta如何修改源码 || Q: How to modify the source code of Deta

### DISCUSSION THREAD

**Discussion by fenychn0206** [walinejs/waline#2678] [Q&A] [answered]
### Describe the bug | 问题描述

Deta只能修改环境变量，但是如果要改服务端源码不知道怎么搞（就是部署在vercel的时候可以直接从github上面改源码的那个意思）

### Website URL | 问题网站

https://deta.space

### Where your waline deploy? | 服务部署在哪里？

Deta

### Where your comment data store? | 数据存储在哪里？

Deta Base

---
### Describe the bug | Problem description

Deta can only modify environment variables, but if you want to change the server source code, I don’t know how to do it (that is, when deployed in vercel, you can change the source code directly from github)

### Website URL | Problem website

https://deta.space

### Where your waline deploy? | Where is the service deployed?

Deta

### Where your comment data store? | Where is the data stored?

Deta Base

**Comment by lizheming** [comment, marked as answer]
应用仓库脚手架：https://github.com/walinejs/deta-starter
如何创建新应用：https://deta.space/docs/en/build/new-apps
可以按照文档自己折腾下

---
Application warehouse scaffolding: https://github.com/walinejs/deta-starter
How to create a new app: https://deta.space/docs/en/build/new-apps
You can fiddle with it yourself according to the documentation

**Reply by fenychn0206** [reply]
好的谢谢🙏

---
OK thank you🙏
